schema/schema.py

```diff
@@ -22,20 +30,22 @@ def format_param(param: click.core.Option) -> Dict[str, Union[Optional[str], Lis
                        a list of those allowed options
     * default - The default option for that parameter
     """
-    formatted_param: Dict[str, Union[Optional[str], List[str]]] = {"name": param.name, "help": param.help}
+    formatted_param: Dict[str, Any] = {"title": param.name, "description": param.help}
 
     # NOTE: Params do not have explicit "string" type; either "text" or "path".
     #       All choice options are from a set of strings.
-    if param.type.name in ["text", "path", "choice"]:
+    if param.type.name in ["text", "path", "choice", "filename", "directory"]:
         formatted_param["type"] = "string"
+    elif param.type.name == "LIST":
```

Comment:
We could, though since we are only using the string in this one situation, and the logic is localized to this function, I don't see much of a need for having it separated from this logic.
I wouldn't be opposed to making it constant, I just don't see much reason to right now besides adding abstracting it.